starter/script.js

```diff
@@ -1 +1,153 @@
-// put code here!
+
+/*
+Each path of decisions is inside its own function.
+An array, paths, holds variables representing those functions / paths.
+play() starts the game.  It asks the user for a number. Then a for loop goes
+thru paths and calls a function at the index matching the number entered by the
+user.At the end of each path, the user is asked if she wants to play again.
+That gets done by the playAgain() function. If user answers yes, play() starts
+the game again.
+*/
+
+var paths = [null, snacks, bball, fball];
+play();
+
+function play() {
+  alert("You're about to be born again, this time as a pro athlete...\n" +
+        'Answer the following questions to fulfill your destiny...\n' +
+        'Ready? Press Ok to start');
+  var intensity = parseInt(prompt('On a scale of 1 - 3, how intense are you in your current life...?\n' +
+                                  '(3 === most intense)'));
+  for (var i = 1; i < paths.length; i++) {
+    if (i === intensity) {
+      paths[i]();
+    }
+  }
+}
+
+function fball() {
+  var qDance = prompt('In this next life, do you like to dance? (y/n)');
+  if (qDance == 'y') {
+    var qFast = prompt('So, you think you can dance, and you especially like to dance when you celebrate.  But Do you run really fast? (y/n)');
+    if (qFast == 'n') {
+      var qNorS = prompt('So, it looks like, in this life, you are born in the United States, but choose: north or south? (north/south)');
```

Comment:
I would just watch out with the naming of some of your variables, jumping in and reading your code I found some of them a little difficult to understand the context of what the variable was used for- I understand that `var qNorS`, now represents North or South, but maybe choosing to name it something more clear like `var location`? Just something to think about for the future